src/ILCompiler.Compiler/src/CppCodeGen/ILToCppImporter.cs

```diff
@@ -1652,6 +1776,7 @@ private void ImportStoreIndirect(TypeDesc type)
 
             // TODO: Write barrier as necessary!!!
 
+            AppendLine();
```

Comment:
I'll keep the formatting when ```-nolinenumbers``` is set, and reduce it otherwise to preserve a decent debugging experience when using the C# code.